libraries/AR_WPNav/AR_WPNav.cpp

```diff
@@ -618,7 +630,7 @@ void AR_WPNav::update_speed_max()
     _last_speed_update_ms = now_ms;
 
     // update position controller max speed
-    _pos_control.set_limits(speed_max, _pos_control.get_accel_max(), _pos_control.get_lat_accel_max(), _pos_control.get_jerk_max());
+     _pos_control.set_limits(speed_max, accel_max, lat_accel_max, jerk_max);
```

Comment:
Thanks for this.  Can you remove the whitespace added at the beginning of this line?